src/components/Navbar.js

```diff
@@ -0,0 +1,13 @@
+const Navbar = () => (
+  <nav className="navbar">
+    <h1>Math Magician app</h1>
+    <div className="links-container">
+      <a className="links" href="/home">Home</a>
```

Comment:
- [ ] Here consider changing it like this ti fix the bug `<a className="links" href="/">Home</a>`